engine/apps/api/views/user_notification_policy.py

```diff
@@ -32,18 +25,29 @@
 
 class UserNotificationPolicyView(UpdateSerializerMixin, ModelViewSet):
     authentication_classes = (PluginAuthentication,)
-    permission_classes = (IsAuthenticated, ActionPermission)
-
-    action_permissions = {
-        IsAdminOrEditor: (*MODIFY_ACTIONS, "move_to_position"),
-        AnyRole: (*READ_ACTIONS, "delay_options", "notify_by_options"),
-    }
-    action_object_permissions = {
-        IsOwnerOrAdmin: (*MODIFY_ACTIONS, "move_to_position"),
-        AnyRole: READ_ACTIONS,
+    permission_classes = (IsAuthenticated, RBACPermission)
+
+    rbac_permissions = {
+        # TODO: what permissions should go here?
+        "list": [],
+        "retrieve": [],
+        "create": [],
+        "update": [],
+        "destroy": [],
+        "move_to_position": [],
+        "delay_options": [],
+        "notify_by_options": [],
     }
```

Comment:
@Matvey-Kuk which permission object type should user notification policies fall under?

Reply:
should be a mix of your/teammates notification settings